### PR TITLE
Add outgoing webhook documentation

### PR DESCRIPTION
**Pull request by gaborszakacs**

### CODE REVIEW

**Comment by dbreuer** [comment]
[ ] Please write a descriptions for event types
[ ] Please add explanation for redelivery (delivered once by default)

**Comment by gaborszakacs** [comment]
@dbreuer I'm not sure what do you mean by event type description, I wrote: 

> Currently only build events are supported: a notification is sent when a build starts or finishes.

I've added more explanation for the redelivery action.

File: docs/api/outgoing-webhooks.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  "git": {
+    "provider":"github",
+    "src_branch":"feature/branch",
+    "dst_branch":master, # If the build was triggered by a pull request
+    "pull_request_id":32 # If the build was triggered by a pull request
+  }
+}
+```
+
+The recent delivery attempts of a webhook can be found on the Code tab, so you can inspect the sent payload and the given response, and even request a redelivery of a given item. (Bitrise attempts to deliver an item once unless requested manually.)
```
Comment (viktorbenei) [inline docs/api/outgoing-webhooks.md]: Move this to the top, before the payload description.

Comment (viktorbenei) [reply docs/api/outgoing-webhooks.md]: Also highlight that we don't do automatic retry / will try to deliver the event exactly once.

**Comment by gaborszakacs** [comment]
@viktorbenei updated

**Pull request merged**